mithril-client-cli/src/commands/snapshot/download.rs

```diff
@@ -41,35 +46,66 @@ pub struct SnapshotDownloadCommand {
 impl SnapshotDownloadCommand {
     /// Command execution
     pub async fn execute(&self, config_builder: ConfigBuilder<DefaultState>) -> StdResult<()> {
```

Comment:
code smell: this function is too large by far, containing too many local bindings.

Reply:
I've removed a few unnecessary bindings. A refacto of this function will be made in a next PR.